src/status_im2/contexts/wallet/send/select_asset/style.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im2.contexts.wallet.send.select-asset.style)
+
+(defn container
+  [margin-top]
+  {:flex       1
+   :margin-top margin-top})
```

Comment:
what about insets in screens.cljs file instead?

Reply:
Replaced with SafeAreaView, that seems to achieve same results but with IMO the correct approach  https://reactnative.dev/docs/safeareaview